### PR TITLE
Fix accessibility violations

### PR DESCRIPTION
**Pull request by kaushalnavneet**
Issues:
https://github.ibm.com/org-ids/roadmap/issues/23831#issuecomment-50542297
https://github.ibm.com/org-ids/roadmap/issues/23622
https://github.ibm.com/org-ids/roadmap/issues/23623

### CODE REVIEW

File: packages/components/src/components/Tabs/Tabs.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ import {
 } from 'carbon-components-react';
 
 const TabContent = ({ selected, ...other }) =>
-  selected && <CarbonTabContent selected={selected} {...other} />;
+  selected ? <CarbonTabContent selected={selected} {...other} /> : <span style="visibility: hidden;"></span>;
```
Comment (AlanGreene) [inline packages/components/src/components/Tabs/Tabs.js:22]: Still missing the id on the new span or is it not required?

Comment (kaushalnavneet) [reply packages/components/src/components/Tabs/Tabs.js:22]: Thanks, the `id` attribute was important. I've added the `id` and the violation is gone. (Adding visibility style didn't have any impact but still left it there just to be on the safer side) This is what I get after manipulating the elements from the Chrome dev tool. Let's see if the changes in the package component work as well. 

Screen Shot 2022-11-22 at 2 07 42 PM

**Comment by AlanGreene** [comment]
Flaky unit test unrelated to the changes in this PR, see https://github.com/tektoncd/dashboard/pull/2572#issuecomment-1305870291
/test tekton-dashboard-unit-tests

**Comment by kaushalnavneet** [comment]
Trying to rerun the tests -
/test tekton-dashboard-unit-tests

**Comment by AlanGreene** [comment]
The previous failing test is passing now but it's also failing on the linter:

```
> eslint --ignore-path .gitignore .
/home/prow/go/src/github.com/tektoncd/dashboard/packages/components/src/components/FormattedDate/FormattedDate.js
  63:10  error  Replace `<span·title={formattedDate}·aria-label={formattedDate}>{content}</span>` with `(⏎····<span·title={formattedDate}·aria-label={formattedDate}>⏎······{content}⏎····</span>⏎··)`  prettier/prettier
/home/prow/go/src/github.com/tektoncd/dashboard/packages/components/src/components/Tabs/Tabs.js
  22:14  error  Replace `<CarbonTabContent·selected={selected}·{...other}·/>·:·<span·className="tkn--visually-hidden"·id={other.id}></span>` with `(⏎····<CarbonTabContent·selected={selected}·{...other}·/>⏎··)·:·(⏎····<span·className="tkn--visually-hidden"·id={other.id}></span>⏎··)`  prettier/prettier
  22:68  error  Empty components are self-closing                                                                                                                                                                                                                                           react/self-closing-comp
✖ 3 problems (3 errors, 0 warnings)
  3 errors and 0 warnings potentially fixable with the `--fix` option.
```

You can `npm run lint:fix` to auto-fix these, commit and squash the resulting change and you should be good to go.

**Comment by kaushalnavneet** [comment]
Allow testing -
/ok-to-test

**Comment by kaushalnavneet** [comment]
@AlanGreene it might like your comment to start the tests again. I guess it has to come from one of the approvers.

**Review by AlanGreene — APPROVED** [review]
Thanks @kaushalnavneet 
/lgtm
/meow

**Pull request merged**